_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy.cpp

```diff
@@ -3860,6 +3860,8 @@ mfxStatus Legacy::CheckSlices(
 
     if (bCheckNMB)
     {
+        MFX_CHECK(!IsOn(par.mfx.LowPower), MFX_ERR_UNSUPPORTED);
```

Comment:
It is the case for incompatible_param. The function should not exit immediately, but change pCO2->NumMbPerSlice to zero, ++changed, and continue to check other parameters.
User will now the feature is not supported by seeing NumMbPerSlice has changed to 0.

Reply:
Thanks @lakulako. I will upload a commit to fix that.